platform.txt

```diff
@@ -101,10 +101,6 @@ recipe.output.save_file_hex={build.project_name}.save.hex
 recipe.size.pattern="{compiler.path}{compiler.size.cmd}" -A "{build.path}/{build.project_name}.elf"
 recipe.size.regex=\.text\s+([0-9]+).*
 
-## Export Compiled Binary
-recipe.output.tmp_file={build.project_name}.hex
-recipe.output.save_file={build.project_name}.{build.variant}.hex
```

Comment:
Sorry. I'll do a clean PR, looks like some of my legacy changes made it
into my commit

On 24 Aug 2016 8:14 a.m., "Sandeep Mistry" <notifications@github.com> wrote:

> In platform.txt
> <https://github.com/sandeepmistry/arduino-nRF5/pull/25#discussion_r75975558>
> :
>
> > @@ -101,10 +101,6 @@ recipe.output.save_file_hex={build.project_name}.save.hex
> >  recipe.size.pattern="{compiler.path}{compiler.size.cmd}" -A "{build.path}/{build.project_name}.elf"
> >  recipe.size.regex=\.text\s+([0-9]+).*
> >
> > -## Export Compiled Binary
> > -recipe.output.tmp_file={build.project_name}.hex
> > -recipe.output.save_file={build.project_name}.{build.variant}.hex
>
> I think we need these too? Added in 54028f7
> <https://github.com/sandeepmistry/arduino-nRF5/commit/54028f795b41e4d1e6fb708ddfca429bdd45ea2d>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/sandeepmistry/arduino-nRF5/pull/25/files/ea989aa74991988cf177509430a502df65b7771c#r75975558>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAfrizhSOOZ2GzwQDwBtd7SAJqnkcHqQks5qi4z-gaJpZM4JW8yN>
> .
>